llm_on_ray/common/trainer/default_trainer.py

```diff
@@ -208,7 +208,7 @@ def train(self):
                     if step % logging_steps == 0:
                         loss = loss.item()
                         ppl = math.exp(loss)
-                        epochs = (step + idx * total_steps) / (num_train_epochs * total_steps)
+                        epochs = idx + step / total_steps
```

Comment:
What about update `total_steps` to `steps_per_epoch`? Otherwise it was confusing. Also check line 222 when we report the total steps, that should be steps_per_epoch * num_train_epochs?

Reply:
# change before:
if epoch_num == 3, the epoch field in log will be `0, 0.001, 0.002, 0.003, ..., 1`

# change after:
if epoch_num == 3, the epoch field in log will be `0, 0.003, ..., 1, 1.003, ..., 2, 2.003, ..., 3`

transformers' Trainer also print field info like change after.

Reply:
I understand this. My comment was asking to rename `total_steps` to `steps_per_epoch` to make the code more clear. Also I mean check the code in line 222 `"total_steps": min(max_train_step, total_steps)` which looks incorrect because of the confusing name "total_steps". Also, please update your PR title what are being fixed. In your PR description, please list all the fixes that are included in the PR.

Reply:
Updated.
rename the `total_steps` to `steps_per_epoch`, make the variable more clear.